.changelog/4427.internal.md

```diff
@@ -0,0 +1,14 @@
+go: bump tendermint to 0.35
+
+Configuration changes required due to Tendermint changes:
+
+Flags `consensus.tendermint.p2p.max_num_{inbound,outbound}_peers` have
+been replaced with a single `consensus.tendermint.p2p.max_connections`.
+Flag `consensus.tendermint.state_sync.consensus_node` has been removed,
+as it is no longer required.
+
+Previously, when running a sentry node, the node behind the sentry had to
```

Comment:
Is this possible to auto-configure somehow?

Reply:
The new statesync implementation uses P2P, so it doesn't need this.

Reply:
Was more referring to the sentry stuff.